synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -179,7 +182,10 @@ def format(session):
     # See https://pycqa.github.io/isort/docs/configuration/options.html#force-sort-within-sections
     session.run(
         "isort",
+        "--profile=google",
         "--fss",
+        "--known-local-folder=main",
+        "--treat-all-comment-as-code",
```

Comment:
This will cause a lot of changes in the generated libraries. Please create a doc to discuss the motivation for the change.

Reply:
The changes to this noxfile should also be in `synthtool/gcp/templates/python_mono_repo_library/noxfile.py.j2` for consistency between libraries in the split repositories vs monorepo.

Reply:
I'm going to revert this change, after experimentation, I realized it wouldn't improve the results.